File: listas de exercicio/lista 2/lista 2 sala C/questao 4.c
#include <stdio.h>

int main(){
    int A, B, C;
    printf("Digite o primeiro lado de um triangulo:");
    scanf("%d", &A);
    printf("Digite o segundo lado de um triangulo:");
    scanf("%d", &B);
    printf("Digite o terceiro lado de um triangulo:");
    scanf("%d", &C);

    if (A<(B+C) && B<(A+C) && C<(A+B)){
        if (A==B && B==C){
            printf("É um triangulo equilatero");
        }
        else if (A==C || B==C || A==B){
            printf("É um triangulo isósceles");
       }
        else {
            printf("É um triangulo escaleno");
        }
    } else{
        printf("Os valores dados nao formam um triangulo");
    }
}

File: listas de exercicio/listas 2/lista 2 sala python/questao 4.py
A = float(input("Digite o primeiro lado:"))
B = float(input("Digite o segundo lado:"))
C = float(input("Digite o terceiro lado:"))
if A < B + C and B < A + C and C < A + B:
    if A==B and B==C:
        print("é um triangulo equilatero")
    elif A == B or B == C or C == A:
         print("é um triangulo isoceles")
    else:
        print("é um triangulo escaleno")
else:
    print("Nao forma um triangulo")